data/human/free worlds 0 prologue.txt

```diff
@@ -797,7 +797,7 @@ mission "FW Escort 1B"
 		dialog `You have reached <planet>, but you left part of the convoy behind! Better depart and wait for them to arrive in this star system.`
 	on complete
 		payment 400000
-		event "fw escorts abandon timer"
+		event "fw escorts abandon timer" 1
```

Comment:
Providing no value is the same as putting "1" here. There's no reason to change this.

Reply:
```suggestion
		event "fw escorts abandon timer"
```